diff --git a/Objects/fileobject.c b/Objects/fileobject.c
index a7d64ba..bb4433b 100644
--- a/Objects/fileobject.c
+++ b/Objects/fileobject.c
@@ -708,7 +708,7 @@ _portable_fseek(FILE *fp, Py_off_t offset, int whence)
     case SEEK_END:
 #ifdef MS_WINDOWS
         fflush(fp);
-        if (_lseeki64(fileno(fp), 0, 2) == -1)
+        if (_fseeki64(fp, 0, 2) != 0)
             return -1;
 #else
         if (fseek(fp, 0, SEEK_END) != 0)
